src/components/button-toggle/button-toggle.stories.tsx

```diff
@@ -378,11 +378,12 @@ export const DisabledGroup: Story = () => (
 DisabledGroup.storyName = "Disabled Group";
 
 export const WrappedButtons: Story = () => (
-  <Box width="350px" display="flex" flexWrap="nowrap">
+  <Box width="375px" display="flex" flexWrap="nowrap">
```

Comment:
comment (non-blocking): we could set the width to be smaller [when run in chromatic](https://github.com/Sage/carbon/blob/ae5079cf2afaff11554a5562e8f0659bb8bcbab0/.storybook/isChromatic.js) here so we get a snapshot of them stacked fully vertically

![image](https://github.com/user-attachments/assets/b42c17b0-0941-462e-bd4c-7d570f8abf58)


```
 const inChromatic = isChromatic();

  <Box width={inChromatic ? "175px" : "375px"} display="flex" flexWrap="nowrap">

Reply:
alternatively you could add a test story and capture the example you've screenshotted in the above PR description if you prefer that